app/main.ipynb

```diff
@@ -2,17 +2,15 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 18,
```

Comment:
Data extraction: get the data from 3 tables & combine it into single .csv file. After that read this file using pandas to create Dataframe. So it will be all joined data in 1 dataframe. Quick check - should be 74818 rows in it.

You're having 3 files instead of just 1